src/internal/easee/client.go

```diff
@@ -344,6 +431,56 @@ func (c *client) bearerTokenHeader(authToken string) string {
 	return "Bearer " + authToken
 }
 
+func (c *client) checkCommand(r commandResponse) error {
+	ticker := time.NewTicker(c.commandCheckInterval)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			if err := c.runCheck(r); err == nil {
+				return nil
+			}
+		case <-time.After(c.commandCheckTimeout):
+			return errors.New("command checker timeout")
+		}
+	}
+}
+
+func (c *client) runCheck(r commandResponse) error {
```

Comment:
you really cannot do requests to their API every 20ms... At least 300-500ms.

Reply:
yeah, will increase the interval here.